SAP/luigi
# Handle error on core level

Fixes #2148 

How to test:
```
onRuntimeErrorHandler: {
   errorFn: (error)=>{
      console.log('for this node you get an error ', error);
   }
}
```
Add this to a node and force an runtime error on this node.
In browser console you should see the console log
